pkgs/tools/graphics/ldgallery/viewer/node-composition.nix

```diff
@@ -0,0 +1,17 @@
+# This file has been generated by node2nix 1.8.0. Do not edit!
```

Comment:
I think we are avoiding to create self-contained Node packages for a shared approach: https://github.com/NixOS/nixpkgs/pull/102552#discussion_r528206040

Reply:
Sadly, it's not possible to put the package in the shared `pkgs/development/node-packages/node-packages.json` because:
* the node project is not located at the root of the repository (https://github.com/svanderburg/node2nix/issues/177)
* it requires extra development dependencies

(see [comment in `generate.sh`](https://github.com/NixOS/nixpkgs/blob/5d33216756e90a3908b922f3b58b9a5499381e0a/pkgs/tools/graphics/ldgallery/viewer/generate.sh#L4-L6))